app/views/splash/index.html.erb

```diff
@@ -3,11 +3,17 @@
 <div class="container homepage-container">
   <div class= "row">
     <div class="col-sm-6 pt-40 homepage-left">
-      <h1>Shared Research Repository</h1>
-      <h3 class="splash-page">for cultural and heritage organisations</h3>
-      <h4 class="pt-40 splash-page">
-        Welcome to our shared repository where you’ll find research produced by staff of the British Library and our current partner organisations: the British Museum, MOLA (Museum of London Archaeology), National Museums Scotland and Royal Botanic Gardens, Kew.<br/><br/>
-        Visit one repository at a time, or use the search box to search and explore publications, reports, datasets and papers from across our combined content.
+      <div class="flex">
+    <%= image_tag('bannerlogo.png', class: "tenant-logo banner-logo", alt: "Shared Research Repository") %>      
```

Comment:
The indent for this should be 2 spaces further indented than the above line.  This helps scan the HTML nesting.